BedrockNode.cpp

```diff
@@ -67,7 +63,6 @@ BedrockNode::BedrockNode(const SData& args, BedrockServer* server_)
     SINFO("BedrockNode constructor");
 }
 
-// --------------------------------------------------------------------------
```

Comment:
I personally really dislike this type of noise in the code and would love to see it vanish from everywhere in our code base.

To me it provides no visual aid and no other benefit, so it equivalent to [this](https://github.com/Expensify/Web-Expensify/blob/2ff4d91791038977106d7951601b60b4e5f403ea/site/deprecated/policy3.js#L3616-L3623)